pkg/virt-handler/vm.go

```diff
@@ -2047,6 +2053,10 @@ func (d *VirtualMachineController) processVmUpdate(origVMI *v1.VirtualMachineIns
 			if err != nil {
 				return fmt.Errorf("failed to adjust resources: %v", err)
 			}
+		} else if vmi.IsRunning() && !vmi.IsFinal() {
```

Comment:
you don't need the `!vmi.IsFinal()` check here. `vmi.IsRunning()` is enough.

Reply:
Done